core/dbt/artifacts/resources/v1/model.py

```diff
@@ -31,3 +31,9 @@ class Model(CompiledResource):
     latest_version: Optional[NodeVersion] = None
     deprecation_date: Optional[datetime] = None
     defer_relation: Optional[DeferRelation] = None
+
+    def __post_serialize__(self, dct: Dict, context: Optional[Dict] = None):
+        dct = super().__post_serialize__(dct, context)
+        if context and context.get("artifact") and "defer_relation" in dct:
```

Comment:
Is this strictly a refactor or would we be starting to include the `defer_relation` in the programmatic response from `dbtRunner` invocations with this change?

Reply:
This just maintains current functionality, which is that when generating artifacts we remove "defer_relation" and "call_config_dict" from all nodes. We could easily change it now if we want, but in order to minimize disruption I went with preserving the way it currently works.

Reply:
Both of those are included in the msgpack versions (for internal use/partial parsing)